### DISCUSSION TITLE
Leptos server signals?

### DISCUSSION THREAD

**Discussion by derrickquinn** [leptos-rs/leptos#1130] [Ideas]
I was looking for a way to use websockets in my Leptos app, and found [this cool repo](https://github.com/tqwewe/leptos_server_signal ) that offers a nice solution. Is there a plan to do something similar directly within Leptos? It seems very in-line with server functions, and IMO this sort of easy integration with the backend is one of the best reasons to use a WASM frontend in the first place.

**Comment by gbj** [comment]
I've added it to our [`awesome-leptos`](https://github.com/leptos-rs/awesome-leptos) list. I think we're still trying to figure out what streaming or long-lived server functions would look like, if they belong in the framework core itself. This looks like a very useful utility in the meantime!
